src/rules/no-empty-source/README.md

```diff
@@ -0,0 +1,33 @@
+# no-empty-source
+
```

Comment:
Sorry to mess you around, but you still need the "Disallow empty source." bit of the line here.

All the rule READMEs adhere [to the same structure](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#write-the-readme).